src/app/cpe/page/page.component.html

```diff
@@ -0,0 +1,6 @@
+<div [id]="pageId" style="border: brown solid 2px; height: 50vh; width: 80%; margin: 20px" *ngIf="page">
+  <div style="font-weight: bold">{{page.label}}</div>
```

Comment:
no inline styles

Reply:
Wrong here and in our case.  This is a mock component, this is about routing. The decision to make it inline was simply to not pollute or use any scss if the component gets removed or rewritten. I do not wan't to have any scss files in my repository containing sth. which might be used or not. Furthermore we are using tailwind css - no scss. Inline styles are the best option there. In this very case and concerning future use of tw one simply does not want to use any scss. Does our UX have any styles defined yet? C'mon please.